### DISCUSSION TITLE
Tamanho do app muito grande ao instalar a dependência flutter_zoom_videosdk

### DISCUSSION THREAD

**Discussion by thiagodeavila96** [Flutterando/forum#197] [Perguntas e Respostas]
Boa tarde pessoal,
Ao instalar a dependência flutter_zoom_videosdk na aplicação, no momento em que o bundle é gerado, ele acaba ficando muito grande por conta da biblioteca.
Alguém poderia por favor auxiliar em como diminuir?
Obrigado.

**Comment by thiagodeavila96** [comment]
UP

**Comment by maurojohann** [comment]
Acho que vai conseguir encontrar alguma informação no forum da zoom:

https://devforum.zoom.us/t/apk-size-after-adding-android-zoom-sdk/16573/33
